src/sentry/static/sentry/app/components/todos.jsx

```diff
@@ -0,0 +1,312 @@
+      {
+        'task': 3,
+        'title': t('Invite team member'),
+        'description': t('Bring your team aboard'),
+        'skippable': false,
+        'prereq': [],
+        'feature_location': 'organization',
+        'location': 'members/',
+      },
+      {
+        'task': 4,
+        'title': t('Add a second platform'),
+        'description': t('Add Sentry to a second platform'),
+        'skippable': false,
+        'prereq': [1, 2],
+        'feature_location': 'organization',
+        'location': 'projects/new/',
+      },
+      {
+        'task': 5,
+        'title': t('Add user context'),
+        'description': t('Know who is being affected by crashes'),
+        'skippable': false,
+        'prereq': [1, 2],
+        'feature_location': 'absolute',
+        'location': 'https://docs.getsentry.com/hosted/learn/context/#capturing-the-user',
+      },
+      {
+        'task': 6,
+        'title': t('Set up release tracking'),
+        'description': t('See what releases are generating errors.'),
+        'skippable': false,
+        'prereq': [1, 2],
+        'feature_location': 'project',
+        'location': 'release-tracking',
+      },
+      {
+        'task': 7,
+        'title': t('Upload sourcemaps'),
+        'description': t('Deminify javascript stacktraces'),
+        'skippable': false,
+        'prereq': [1, 2, 8], // Is one of the platforms javascript?
+        'feature_location': 'absolute',
+        'location': 'https://docs.getsentry.com/hosted/clients/javascript/sourcemaps/'
+      },
+      // {
+      //   'task': 8,
+      //   'title': 'User crash reports',
+      //   'description': 'Collect user feedback when your application crashes',
+      //   'skippable': false,
+      //   'prereq': [1, 2, 5]
+      //   'feature_location': 'project',
+      //   'location': 'settings/user-reports/'
+      // },
+      {
+        'task': 9,
+        'title': t('Set up issue tracking'),
+        'description': t('Link to Sentry issues within your issue tracker'),
+        'skippable': true,
+        'prereq': [1, 2],
+        'feature_location': 'project',
+        'location': 'issue-tracking',
+      },
+      {
+        'task': 10,
+        'title': t('Set up a notification service'),
+        'description': t('Receive Sentry alerts in Slack or HipChat'),
+        'skippable': true,
+        'prereq': [1, 2],
+        'feature_location': 'project',
+        'location': 'notification',
+      },
+    ]
+  },
+
+  getInitialState() {
+    return {
+      tasks: [],
+      seeAll: false,  // Show all tasks, included those completed
+    };
+  },
+
+  componentWillMount() {
+    let org = this.getOrganization();
+    let tasks = [];
+
+    for (let task of TodoList.TASKS) {
+      task.status = '';
+
+      for (let server_task of org.onboardingTasks) {
+        if (server_task.task == task.task) {
+          task = $.extend(task, server_task);
+          break;
+        }
+      }
+      tasks.push(task);
```

Comment:
It's a little nicer to use `Array.prototype.map`, since you communicate intent up front:

```javascript
let tasks = TodoList.TASKS.map((task) => {
  // ... do stuff ...
  return task;
});
```